src/algebra/char_p/basic.lean

```diff
@@ -242,11 +248,6 @@ theorem sub_pow_char_pow [comm_ring R] {p : ℕ} [fact p.prime]
   (x - y) ^ (p ^ n) = x ^ (p ^ n) - y ^ (p ^ n) :=
 sub_pow_char_pow_of_commute _ _ _ (commute.all _ _)
 
-lemma eq_iff_modeq_int [ring R] (p : ℕ) [char_p R p] (a b : ℤ) :
```

Comment:
Was this a duplicate?

Reply:
Yes, with `int_cast_eq_int_cast`

Reply:
Can you mention that in the PR description?